### PR TITLE
SKALE-3008 updated dependencies to modern, sane and supported versions

### PR DESCRIPTION
**Pull request by sergiy-skalelabs**
**IMA Agent** support for big values in `chainId`, applied correct dependency versions which work:
        "ethereumjs-tx": "1.3.7",
        "ethereumjs-wallet": "0.6.3",
        "ethereumjs-util": "6.1.0",
        "web3": "1.2.1",
Added some low level logging when log level is highest/trace
**ALL SKALE TEST** works

### CODE REVIEW

File: proxy/truffle-config.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
 require('dotenv').config();
 require("ts-node/register");
 const Web3 = require('web3');
-let hdwalletProvider = require('truffle-hdwallet-provider');
+let hdwalletProvider = require('@truffle/hdwallet-provider');
 let schainName = process.env.CHAIN_NAME_SCHAIN;
 let schainRpcUrl = process.env.URL_W3_S_CHAIN;
 let mainnetRpcUrl = process.env.URL_W3_MAIN_NET;
 
+console.log( "URL_W3_MAIN_NET                  =", process.env.URL_W3_MAIN_NET                  );
```
Comment (payvint) [inline proxy/truffle-config.js]: I think it could be better to remove printing env variables

File: npms/skale-ima/index.js [hunk 2]
```diff
@@ -933,6 +949,7 @@ async function receive_eth_payment_from_s_chain_on_main_net(
         const tx = compose_tx_instance( strLogPrefix, {
             chainId: cid_main_net,
             nonce: tcnt,
+            // gas: 8000000, // xxx
```
Comment (payvint) [inline npms/skale-ima/index.js]: Maybe remove comments of the same line

**Review by payvint — CHANGES_REQUESTED** [review]
Need some little changes just to remove comments and printing env variables

File: proxy/truffle-config.js (hunk 1 above)
Comment (sergiy-skalelabs) [reply proxy/truffle-config.js]: sure, removed

File: npms/skale-ima/index.js (hunk 2 above)
Comment (sergiy-skalelabs) [reply npms/skale-ima/index.js]: sure, removed

**Review by payvint — APPROVED** [review]
Good!

**Review by kladkogex — APPROVED** [review]

**Pull request merged**